### ISSUE TITLE
unable to determine selected conversations in gmail using talkback

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#398]
```
What steps will reproduce the problem?
1. select a conversation in gmail.
2. Navigate to another conversation by swiping or explore by touch.
3. Use swiping or explore by touch to find the conversation selected. The user 
has no feedback to find the selected conversation.

What is the expected output? What do you see instead?
I expect talkback to notify me which conversations have been selected.  Instead 
I have no way to determine what content is selected.


What version of the product are you using? On what operating system?
Talkback 3.5.2 Android 4.4.2 gmail version 4.9 (1271612)

Please provide any additional information below.
Until recently, it was possible to tell the number of conversations selected, 
but that info was removed with a gmail update.




```

Original issue reported on code.google.com by `hittsj...@gmail.com` on 1 Oct 2014 at 11:17
